Phonebook.Frontend/src/app/pages/settings/settings.component.ts

```diff
@@ -34,10 +35,22 @@ export class SettingsComponent implements OnInit, OnDestroy {
 
   public ngOnInit() {
     this.languageValue = this.languageService.getLanguage();
+    this.useFirstAprilTheme();
```

Comment:
this will only apply if the settings component is loaded. Thus not on the app startup. You should add it to the main component.